specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/billingSavingsPlansList.json

```diff
@@ -0,0 +1,255 @@
+{
+  "parameters": {
+    "billingAccountName": "00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+    "api-version": "2023-04-01",
+    "$filter": "billingProfileId eq '/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/billingProfiles/AAAA-BBBB-CCC-DDD'",
+    "$orderby": "properties/displayName asc",
+    "refreshSummary": "true",
+    "selectedState": "Succeeded",
+    "take": "3"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/savingsPlanOrders/20000000-0000-0000-0000-000000000000/savingsPlans/30000000-0000-0000-0000-000000000000",
+            "name": "30000000-0000-0000-0000-000000000000",
+            "type": "microsoft.billing/billingAccounts/savingsPlanOrders/savingsPlans",
+            "sku": {
+              "name": "Compute_Savings_Plan"
+            },
+            "properties": {
+              "billingScopeId": "/subscriptions/10000000-0000-0000-0000-000000000000",
+              "billingAccountId": "/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+              "billingProfileId": "/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/billingProfiles/AAAA-BBBB-CCC-DDD",
+              "billingPlan": "P1M",
+              "commitment": {
+                "grain": "Hourly",
+                "currencyCode": "USD",
+                "amount": 10.0
+              },
+              "renew": false,
+              "appliedScopeType": "Shared",
+              "displayName": "SP1",
+              "provisioningState": "Succeeded",
+              "term": "P1Y",
+              "displayProvisioningState": "Succeeded",
+              "userFriendlyAppliedScopeType": "Shared",
+              "expiryDateTime": "2023-12-16T02:17:03.7396152Z",
+              "purchaseDateTime": "2022-12-16T02:14:58.2304134Z",
+              "benefitStartTime": "2022-12-16T02:17:03.7396152Z",
```

Comment:
For consistency, this should be benefitStartDateTime

Reply:
We have been using benefitStartTime in reservation and savings plan: 
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/reservations/resource-manager/Microsoft.Capacity/stable/2022-11-01/reservations.json#L1808

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json#L998